feat: todo-list를 생성하라

wenodev opened the pull request:
- [ ] 현재 로직 확인하기
- [ ] Spring Security 학습하기

wenodev committed:
feat: todo-list를 생성하라

johngrib reviewed (commented):
오래간만입니다. 이번주도 화이팅입니다.

wenodev committed:
feat: Spring Security 의존성을 추가하라

johngrib reviewed (commented):
한국어 명령형 커밋 메시지가 읽기 좋습니다. (제가 좋아하는 스타일입니다)

wenodev committed:
feat: todo-list 작성하라

wenodev committed:
test: 로그인 페이지 비활성화 테스트하라

wenodev committed:
test: csrf 비활성화 테스트하라

wenodev committed:
teat: Servlet의 HandlerInterceptor를 대체할 filter를 생성하라

주어진 코드는 servlet의 HandlerInterceptor interface를 사용하여 인증처리를 했다. 앞으로는 security의 BasicAuthenticationFilter class를 사용하여 시작한다.

wenodev committed:
feat: doFilterInternal를 사용하여 security chain에 끼어들도록 하라

wenodev committed:
fix: Interceptor 비활성화하라

wenodev committed:
test: access token 없을경우 테스트를 통과하라

wenodev commented:
https://github.com/wenodev/TIL/blob/master/retrospect/210310.md 정리하며 학습하고 있습니다.

johngrib reviewed (commented):
99cb81ac220673c691eb6d592c9be4adf03b7710 커밋 메시지가 아쉽습니다.
`test: csrf 비활성화 테스트하라`를 읽으면 마치 테스트코드를 작성한 것 같은데 실제 내용은 그렇지 않아요.

johngrib reviewed (commented):
ee704e25f006202783a72f7e5fd7cade75ef4fc3 커밋 메시지의 본문 설명이 좋습니다. 그러나 어떠한 종류의 의사결정이 있었는지 더 자세히 작성하시면 더 좋겠어요.

johngrib reviewed (commented):
e3d5411608396c1006547501348a08728c26fedc 어떤 인터셉터를 비활성화 한 것인지, 무엇 때문에 그렇게 한 것인지에 대해 커밋 메시지 본문에 작성해 주시면 좋았을 거에요.

wenodev commented:
> [ee704e2](https://github.com/CodeSoom/spring-week7-assignment-1/commit/ee704e25f006202783a72f7e5fd7cade75ef4fc3) 커밋 메시지의 본문 설명이 좋습니다. 그러나 어떠한 종류의 의사결정이 있었는지 더 자세히 작성하시면 더 좋겠어요.
넵!

wenodev commented:
> [99cb81a](https://github.com/CodeSoom/spring-week7-assignment-1/commit/99cb81ac220673c691eb6d592c9be4adf03b7710) 커밋 메시지가 아쉽습니다.
> `test: csrf 비활성화 테스트하라`를 읽으면 마치 테스트코드를 작성한 것 같은데 실제 내용은 그렇지 않아요.

fix : csrf 비활성화하라
이렇게 작성하면 괜찮은 커밋 메세지 일까요?

wenodev commented:
> [e3d5411](https://github.com/CodeSoom/spring-week7-assignment-1/commit/e3d5411608396c1006547501348a08728c26fedc) 어떤 인터셉터를 비활성화 한 것인지, 무엇 때문에 그렇게 한 것인지에 대해 커밋 메시지 본문에 작성해 주시면 좋았을 거에요.

커밋 메세지를 남길때 본문에 현재상황, 작업내용을 작성해야겠어요!

wenodev commented:
이번주 내내 야근을하다보니 학습을 많이 못했네요ㅜㅜ 아직 한달이나 더 남았으니 이런경우도 학습을 할 수 있도록 노력해야겠어요ㅜㅜ

johngrib commented:
> > [99cb81a](https://github.com/CodeSoom/spring-week7-assignment-1/commit/99cb81ac220673c691eb6d592c9be4adf03b7710) 커밋 메시지가 아쉽습니다.
> > `test: csrf 비활성화 테스트하라`를 읽으면 마치 테스트코드를 작성한 것 같은데 실제 내용은 그렇지 않아요.
> 
> fix : csrf 비활성화하라
> 이렇게 작성하면 괜찮은 커밋 메세지 일까요?

네. 조금 더 나은 커밋 메시지가 되었습니다.

johngrib reviewed (commented):
어느새 일요일이 되었군요. 주간 회고를 작성해 공유해 주세요.

johngrib reviewed (changes requested):
PR target 이 `CodeSoom:main`으로 잘못 설정되어 있습니다. 수정해주세요.
![image](https://user-images.githubusercontent.com/1855714/111168668-a0665c80-85e5-11eb-93f1-b4153b6212e4.png)
